modules/deploy/pages/deployment-option/cloud/api/cloud-api-overview.adoc

```diff
@@ -0,0 +1,158 @@
+= Redpanda Cloud API Overview
+:description: Learn about using the Cloud API to manage clusters and cloud resources.
+:page-cloud: true
+:page-api: cloud
+:page-api-slot: overview
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management.
+
+== Requirements
+
+To be able to use the Cloud API:
+
+* You must be a customer with an existing organization in Redpanda Cloud.
+* You can only use one organization for authentication. See xref:./cloud-api-authentication.adoc[] for steps to authenticate API requests.
+
+== Cloud API versus other Redpanda APIs
+
+HTTP Proxy is an HTTP server that exposes some operations you can perform directly on a Redpanda cluster. You can use the Redpanda HTTP Proxy API to perform a subset of actions that are available through the Kafka API, using simpler REST operations.
+
+The Redpanda Schema Registry API is a REST API to manage schemas within a Redpanda cluster.
+
+The Redpanda Admin API is a REST API to manage aspects of a Redpanda cluster. You can use the Admin API to perform operations that are specific to the Redpanda platform and cannot be done using the standard Kafka API.
+
+Redpanda Cloud uses a control plane and data plane architecture. The *control plane* is where most cluster management, operations, and maintenance takes place. The control plane enforces rules in the data plane. The *data plane* is where your cloud cluster lives. The term _data plane_ is used interchangeably with _cluster_. 
+
+The *Control Plane API* allows you to interact with the control plane. You use the Control Plane API to manage cloud clusters, organization, resource groups, networks, and other cloud resources. The Control Plane API does not interact directly with Redpanda clusters, although some of the actions performed through the API can make their way eventually to a cluster.
+
+The *Data Plane APIs* are a collection of APIs through which you can perform actions against a Redpanda cluster. These APIs expose more high-level operations on the data plane by acting as a wrapper around the various APIs mentioned above that are supported by Redpanda (Kafka API, Admin API, or Schema Registry API). 
+
+== Base URLs
```

Comment:
Sent you the change in Slack because what I see here and in the preview looks different to me